ibis/pandas/tests/test_udf.py

```diff
@@ -271,6 +274,88 @@ def my_mean(series):
     tm.assert_frame_equal(result, expected)
 
 
+def test_udaf_window_interval():
```

Comment:
are all of the window udf tests here or in test_window?

Reply:
all winndow udf tests are here